distros/spring/pom.xml

```diff
@@ -51,6 +51,14 @@
             <groupId>com.connexta.transformation</groupId>
             <artifactId>transformation-api-rest-models</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.connexta.transformation</groupId>
+            <artifactId>commons-api</artifactId>
```

Comment:
You can now delete the other `commons-api` dependency that includes the hard-coded version from my PR below

Reply:
Please. Let's make sure that versions are only defined in the dependency management section of the parent pom using properties.